src/goto-programs/goto_trace.h

```diff
@@ -118,6 +118,9 @@ class goto_trace_stept
   bool cond_value;
   exprt cond_expr;
 
+  // for assignments
```

Comment:
...and declarations (lhs only)

Reply:
Suggest naming these `ssa_lhs/rhs` to make clear what distinguishes them from `full_lhs` and `full_lhs_value` below.

Reply:
Good points, all done as suggested!